src/components/Tables/TableSurface.tsx

```diff
@@ -10,7 +10,7 @@ const StyledPaper = styled(Paper)`
     border-radius: 1rem;
 `;
 
-interface Props {
+export interface Props {
```

Comment:
For something this generically named and small, I'd rather you just redefine it where you need it. Good job reusing code, but this is a case where we should keep it simple